Rakefile

```diff
@@ -3,3 +3,21 @@ begin
   require 'rspec-system/rake_task'
 rescue LoadError
 end
+
+require 'puppet-lint/tasks/puppet-lint'
+PuppetLint.configuration.send('disable_80chars')
+PuppetLint.configuration.relative = true
+PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+
+desc 'Validate manifests, templates, and ruby files'
```

Comment:
There's a [validate task in puppetlabs_spec_helper](https://github.com/puppetlabs/puppetlabs_spec_helper/blob/master/lib/puppetlabs_spec_helper/rake_tasks.rb). Why not use that?